### ISSUE TITLE
[LINUX] Problems with US international keyboard layout

### ISSUE THREAD

**Issue by gdmsl** [ValveSoftware/Dota-2#268]
I cannot have bindings in game for the keys ' and `  when using the us international layout (us-acentos) where ' and ` are two modifier keys to get acentos on various letters. For example ` + e will give è and the character ` is obtained with ` + space. In the old client I had the ` assigned to the scoreboard and it worked.

xev gives this output when pressing the key:

```
KeyPress event, serial 32, synthetic NO, window 0x3a00001,
    root 0x1d1, subw 0x0, time 46642086, (280,197), root:(968,222),
    state 0x0, keycode 0 (keysym 0x60, grave), same_screen YES,
    XKeysymToKeycode returns keycode: 49
    XLookupString gives 0 bytes: 
    XmbLookupString gives 1 bytes: (60) "`"
    XFilterEvent returns: False
```

**Comment by gdrewb-valve** [comment]
SDL is not generating key up/down events for this key as it's part of the accent composition.  I've opened SDL bug #3076 for this to see if they can change their behavior.

**Comment by gdrewb-valve** [comment]
Is this still happening after the March 24 SDL update?

**Comment by gdrewb-valve** [comment]
Checking one last time, is this still happening?

**Comment by gdmsl** [comment]
yes it is still happening. I tried to bind the ` or the ' to an action but the client ignored it keeping the original binding.

**Comment by gdrewb-valve** [comment]
If you add -input_message_spew to your launch options and look at console output in a terminal (or via a file using -con_logfile) when you press those keys what does it say?

**Comment by gdmsl** [comment]
```
SDL:  got SDL_KEYDOWN
KEYDOWN (keyc): sdlkeycode=1073741824 sdlscancode=53 SDL_GetKeyName='' buttoncode=0 buttonname='' ButtonCode_GetLocalKeyNameUTF8=''
SDL:  got SDL_KEYUP
KEYUP (keyc) sdlkeycode=1073741824 sdlscancode=53 buttoncode=0 buttonname=''
```

when pressing ``` button

**Comment by gdrewb-valve** [comment]
OK, the SDL update didn't have an impact so we'll leave it assigned to the SDL dev.

**Comment by gdrewb-valve** [comment]
Are you still seeing this problem?

**Comment by gdmsl** [comment]
Yes. Just tried. The problem persists.
